tools/test/CoverageOutputGenerator/java/com/google/devtools/coverageoutputgenerator/Main.java

```diff
@@ -88,6 +114,23 @@ public static void main(String[] args) {
     System.exit(exitStatus);
   }
 
+  /**
+   * Copies the content of the source file into the destination file.
+   *
+   * <p> Returns true or false depending whether the copy operation was successful or not.
+   */
+  private static boolean copy(File source, File dest) {
```

Comment:
Is this significantly faster than java.io.file.Files.copy() ?

Reply:
No I actually forgot about Files.copy. According to https://baptiste-wicht.com/posts/2010/08/file-copy-in-java-benchmark.html when copying big binaries most common methods take almost the same time.